impeller/renderer/backend/gles/test/pipeline_library_gles_unittests.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/fixtures/box_fade.frag.h"
+#include "impeller/fixtures/box_fade.vert.h"
+#include "impeller/playground/playground_test.h"
+#include "impeller/renderer/backend/gles/pipeline_gles.h"
+#include "impeller/renderer/backend/gles/pipeline_library_gles.h"
+#include "impeller/renderer/pipeline_library.h"
+
+namespace impeller::testing {
+
+using PipelineLibraryGLESTest = PlaygroundTest;
+INSTANTIATE_OPENGLES_PLAYGROUND_SUITE(PipelineLibraryGLESTest);
+
+TEST_P(PipelineLibraryGLESTest, ProgramHandlesAreReused) {
```

Comment:
Nit: would be nice to have a spec constant unit test here too, it would make future errors easier to track down.

Reply:
Done.